07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,33 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Profession Portal</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <header>
+    <h1>Profession Portal</h1>
+    <div class="header_bar">
+      <a href="#">Home</a>
+      <a href="#">Companies</a>
+      <a href="#">Positions</a>
+    </div>
+  </header>
+  <main>
+    <section class="box1">
+      <h2>Hire an employee</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        <br>
+        <br>
+      <a href="#">Sign up as company</a>
+    </section>
+    <section class="box2">
+      <h2>Looking for a job</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        <br>
+        <br>
+      <a href="#">Sign up as employee</a>
+    </section>
+  </main>
 </body>
 </html>
```

Comment:
Excellent job with this assignment. I really love how you made this responsive. Keep up the good work Andrew!